media/stylus/components/structure/socialaccount-providers.styl

```diff
@@ -105,3 +105,7 @@ a.github-button {
         }
     }
 }
+
+.socialaccount-providers .wait-for-persona, .wait-for-persona {
+    display: none;
```

Comment:
Rather than hiding these please disable them by adding the class `disabled` in the HTML. Then you can make the class your script interacts with `js-wait-for-persona` and not need to add anything to the style sheets.